doc/metrics.md

```diff
@@ -151,12 +155,15 @@ scrape_configs:
       #      is not covered by the certificate (not in the SAN list)
       server_name: 'foo'
 ```
-
```

Comment:
This empty line is required (that's the reason for the failure of the doc linter).